run/common/docker/run.sh

```diff
@@ -191,4 +189,4 @@ case "$1" in
         usage  # Display usage information and exit with error
         exit 1
         ;;
-esac
+esac
```

Comment:
```suggestion
esac

```

(newline at end of file)

Reply:
Adding an issue for a CI check.There are at least 383 files like this. We will fix them all there. https://github.com/cardano-foundation/cardano-wallet/issues/5014